packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -1960,6 +1966,11 @@ export const getRepositoriesByTeam = async (
 
     dashboard.repositories = syncedRepositories.sort(sortByNameAscending);
   } catch (error) {
+    if (!bypassLoading && error?.response?.status === 504) {
```

Comment:
Wondering if `bypassLoading` being false is enough to completely hide errors from the second request, it's probably weird to see an error but have data on the screen. Thoughts?

Reply:
If the user is looking for a certain repository that has yet to be synced but something fails, we probably still want to show an error. We might want to word the error differently though, something like: "There was a problem syncing your repositories with GitHub, data might not be up to date."